compiler/rustc_codegen_ssa/src/mir/debuginfo.rs

```diff
@@ -370,6 +370,11 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 if attrs.flags.contains(CodegenFnAttrFlags::NAKED) {
                     return;
                 }
+                // FIXME: Don't spill scalable simd, this works for most of them however,
+                // some intermediate types can't be spilled e.g. `<vscale x 4 x i1>`
```

Comment:
I'm pretty sure even with this change, we do end up putting every value that cg_ssa couldn't determine as having a single assignment on the stack anyway.